elasticsearch/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM elasticsearch:6.8.6
+FROM elasticsearch:8.6
```

Comment:
Should probably be 7.6, not 8.6. https://devdocs.magento.com/guides/v2.4/install-gde/prereq/elasticsearch.html